validators/articles/index.js

```diff
@@ -92,5 +92,22 @@ export default {
         message: e,
       });
     }
+  },
+
+  addComment: async (req, res, next) => {
+    const rules = {
+      content: 'required|string'
```

Comment:
I think there should also be a validation for the article slug coming from the request params object in case its not provided for or is not a string

Reply:
Hi @caleb-42, thank you for your feedback, I made use of the `articleSlug` validation which exists in the validations to check for the article slug coming from the request params object.

Reply:
ok. nice